src/detections/chart/use-data.jsx

```diff
@@ -0,0 +1,13 @@
+import { labels } from 'detections/chart/labels.json'
+import { byMode } from 'detections/chart/by-mode'
+import { buildDatasets } from 'detections/chart/build-datasets'
+import { useItems } from 'detections/chart/use-items'
+
+export const useData = mode => {
```

Comment:
Let's change this file's extension to `.js`.

Reply:
done